config/config.go

```diff
@@ -76,14 +74,12 @@ func Get() (*Configuration, error) {
 		DefaultMaxLimit:            1000,
 		DefaultOffset:              0,
 		KafkaConfig: KafkaConfig{
+			Brokers:                               brokers,
```

Comment:
Why is this separated out? It's convenient for the actual default value to be mentioned here to be consistent with all the other configs

Reply:
```suggestion
			Brokers:                               []string{"localhost:9092"},
```
Add remove line 58